### PR TITLE
135 fix data in `nest_basic_teal` app

### LINKED ISSUE #135: [Bug]: `{nest_basic-teal_dev}` `Error: object 'IRIS' not found`
### What happened?

Go here https://genentech.shinyapps.io/nest_basic-teal_dev/ and click `Show R Code`. `IRIS` used in the code is written with capital letters, the same as `MTCARS` (instead of `iris` or `mtcars`). This brings an error when you run this code in your local session

image

```r
warning('Code was not verified for reproducibility.')
# Add any code to install/load your NEST environment here

library(shiny)
library(teal.code)
library(teal.data)
library(teal.slice)
library(teal)

message('Preprocessing is empty')

stopifnot(rlang::hash(IRIS) == "34844aba7bde36f5a34f6d8e39803508")
stopifnot(rlang::hash(MTCARS) == "d0487363db4e6cc64fdb740cb6617fc0")

IRIS <- dplyr::filter(IRIS, Species == "setosa")
```

```r
Error: object 'IRIS' not found
```

### Relevant log output

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct.

### Contribution Guidelines

- [X] I agree to follow this project's Contribution Guidelines.

### Security Policy

- [X] I agree to follow this project's Security Policy.

### PR DESCRIPTION
**Pull request by m7pr**
Close #135

### CODE REVIEW

File: basic-teal/app.R [hunk 1]
```diff
@@ -3,7 +3,7 @@ library(teal)
 nest_logo <- "https://raw.githubusercontent.com/insightsengineering/hex-stickers/main/PNG/nest.png"
 
 app <- init(
-  data = list(IRIS = iris, MTCARS = mtcars),
+  data = teal.data::teal_data(IRIS = iris, MTCARS = mtcars, code = c("IRIS <- iris;MTCARS <- mtcars")),
```
Comment (pawelru) [inline basic-teal/app.R:6]: do you have to specify `teal.data::`?

**Comment by m7pr** [comment]
We either use `teal.data::teal_data` or we use lowercase letters instead of capitalized IRIS and MTCARS

**Comment by vedhav** [comment]
Please target the change to `dev`

**Comment by pawelru** [comment]
> We either use `teal.data::teal_data` or we use lowercase letters instead of capitalized IRIS and MTCARS

My question was rather: can it be just `teal_data(...)`

**Comment by vedhav** [comment]
Okay, I see that this change is already there in `dev`. So, I think no change is required.

File: basic-teal/app.R (hunk 1 above)
Comment (m7pr) [reply basic-teal/app.R:6]: actually I don't - will remove prefix

**Comment by m7pr** [comment]
> Okay, I see that this change is already there in dev. So, I think no change is required.

So why I am testing outdated https://genentech.shinyapps.io/nest_basic-teal_dev/ ? 
Should we re-deploy before we start testing https://github.com/insightsengineering/coredev-tasks/issues/489

**Comment by m7pr** [comment]
@vedhav would you mind helping to resolve conflicts?

**Comment by vedhav** [comment]
> @vedhav would you mind helping to resolve conflicts?

We'd have to rebase this. Sure, I can do this.

**Comment by vedhav** [comment]
btw, we don't even need a PR in `dev` branch as it's not protected 🤫

**Review by vedhav — APPROVED** [review]
Thanks!

**Pull request merged**